README.md

```diff
@@ -125,8 +144,18 @@ Current state | Next word | Next state
 
 ## A2.4
 
-«replace this with your answer»
-
+Current state | Next word | Next state
+--------------|-----------|-----------
+    S0        |    the    |     S1
+    S1        |<adjective>|     S2
+    S1        |  <noun>   |     S3
+    S2        |<adjective>|     S2
+    S2        |  <noun>   |     S3
+    S3        |  <verb>   |     S4
+    S4        |    "."    |     S5
+    S4        | <adverb>  |     S6
+    S6        |    "."    |     S5
+    S5        |    EOI    |    END
```

Comment:
You have an extra state here: you can combine S1 and S2, so that S1 loops back to itself shen it sees an adjective

# -2